generated/.htaccess

```diff
@@ -1,2 +0,0 @@
-Order deny,allow
-Deny from all
```

Comment:
That, uhm, shouldn't be removed..  deleted the generated folder out of habit I think.